README.md

```diff
@@ -1,279 +1,43 @@
-* `ember deploy` This commands builds your ember application, uploads it to your file hoster and uploads it to your key value store in one step.
-
-You can pass `--environment <some-environment>` to every command. If you don't pass an environment explicitly `ember-deploy` will use the `development`-environment.
-
-You can pass `--deploy-config-file <path/to/deploy-config.js>` to every command. If you don't pass a deploy-config-file explicitly `config/deploy.js` will be read.
-
-## Lightning-Approach Workflow
-
-`ember-deploy` is built around the idea of adapters for custom deployment strategies but for most people the approach Luke suggest in his talk will be a perfect fit.
-
-To use the Lightning-Approach workflow from Luke's talk you will also need to install the redis- and s3-adapters for `ember-deploy`:
+Simply run:
 
 ```
-npm install ember-deploy-redis ember-deploy-s3 --save-dev
+ember install:addon ember-cli-deploy
 ```
 
-__Please watch Luke's Talk before using this project!__
-
-The TL;DR of the talk is that you want to serve your bootstrap index.html that Ember-CLI builds for you from a Key-Value store via your Backend and serve the rest of your assets from a static file hoster like for example [S3](https://aws.amazon.com/de/s3/). This is a sketch of what this Addon gives you out of the box:
-
-![Workflow](https://dl.dropboxusercontent.com/s/tun9kbr4eyrcama/ember-deploy.png?dl=0)
-
-A deployment consists of multiple steps:
-
-1. Build your assets for production via `ember build --environment production`
-2. Upload assets to S3.
-3. Upload your bootstrap index.html to a key-value store.
-4. Activate the uploaded index.html as the default revision your users will get served from your backend.
+## Configuration
 
-If you don't install the redis- and s3-adapters you will need to use a custom adapter you or the community have written. Please have a look at the Custom-Adapters section of this README for further information. In essence your custom adapters will implement the same steps as for the Lightning-Approach but you can customize the different steps as you see fit.
+Configuration is supplied to ember-cli-deploy via a configuration file.  By default, this file should be called `deploy.js` and should be located in the `config` directory in the root of your Ember CLI project.
 
-## Config file
+Alternatively, you can specify a different file location by passing the `deploy-config-file` option to the `ember deploy` command.
 
-By default, `ember-deploy` expects a `deploy.js` file in the `config/` directory of your Ember-CLI project root. In this file you tell `ember-deploy` about the necessary credentials for your file hoster and for your key-value store. An example could look like this:
+An example of the configuration file is as follows:
 
-```js
+```javascript
+// config/deploy.js
 module.exports = {
```

Comment:
Have left the config file example blank until we have a better idea of what it will actually look like.  This is not clear in my head at the moment as I believe it's going to change quite drastically with the idea of the pipeline and plugins.